### PR TITLE
feature: React

### PR DESCRIPTION
**Pull request by rokaskasperavicius**
Moved from Go templates to React for frontend rendering

[http://165.22.79.16:5000/](http://165.22.79.16:5000/) - Staging environment with React running

### CODE REVIEW

File: backend/api/login.go [hunk 1]
```diff
@@ -1,99 +1,90 @@
...
 	if err != nil {
 		log.Println("Error getting session", err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
+
 	for k := range session.Values {
 		delete(session.Values, k)
 	}
+	session.Options.MaxAge = -1
```
Comment (quacklemtduck) [inline backend/api/login.go:79]: What does this do?

**Review by quacklemtduck — APPROVED** [review]
Everything looks good on the backend
Didn't bother looking at the frontend, I trust you on that one

File: backend/api/login.go (hunk 1 above)
Comment (rokaskasperavicius) [reply backend/api/login.go:79]: It removes the cookie by immediately expiring it, as the `delete` for some reason doesn't. Maybe you got a clue why

Comment (rokaskasperavicius) [reply backend/api/login.go:79]: My idea was that the delete just deletes the info in the `Auth` cookie, but it still leaves empty state there. Frontend can't really distinguish between if the Auth cookie has user id or not inside

**Review by Hopeberg — APPROVED** [review]

**Pull request merged**